tests/module_tables.yaml

```diff
@@ -36,7 +36,7 @@ ports:
         order_by: ports.ifIndex, ports.ifDescr, ports.ifName
 cipsec-tunnels:
     ipsec_tunnels:
-        excluded_fields: [tunnel_id, device_id]
+        excluded_fields: [local_addr, tunnel_id, device_id]
```

Comment:
local_addr is either "private" (so not really possible to do any evaluation on it) or the IP address of the snmpsimd (which can change, so no evaluation must be done). Now it is excluded.